### ISSUE TITLE
[WCAG] About Panel - Optimize for consistent styles and layout

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/GeoView#3477]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

- Styles aren't consistent between content output types (markdown vs HTML)
- CSS implementation could be optimized (use theme values rather than hard-coded values)
- All CSS should be placed in the about-panel-style file
